src/sentry/models/notificationsetting.py

```diff
@@ -104,6 +104,11 @@ def provider_str(self) -> str:
         ),
         null=False,
     )
+    is_migrated = models.BooleanField(
+        default=False,
```

Comment:
@snigdhas I think we should just remove `default=False` because I think it might backfill. We should be as safe as possible with this migration, no sense doing anything that does more than run an alter table.

Reply:
Columns with default values are safe to add now. Postgres 10+ doesn't lock the table anymore when adding columns with default values.